tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/addons.py

```diff
@@ -32,11 +34,19 @@ def details_screen_element(self):
         return self.marionette.find_element(*self._details_page_locator)
 
     @property
+    def items(self):
+        elements = self.marionette.find_elements(*self._items_locator)
```

Comment:
We can shorten this function into something smaller:
```python
return [Addon(self.marionette, element) for element in self.marionette.find_elements(*self._items_locator)]
```

This is called a [list comprehension](https://docs.python.org/2.7/tutorial/datastructures.html#list-comprehensions) and we use it at some other places in the page classes